vendor/github.com/containerd/containerd/image.go

```diff
@@ -336,6 +336,14 @@ func WithUnpackDuplicationSuppressor(suppressor kmutex.KeyedLocker) UnpackOpt {
 	}
 }
 
+// WithUnpackApplyOpts appends new apply options on the UnpackConfig.
+func WithUnpackApplyOpts(opts ...diff.ApplyOpt) UnpackOpt {
+	return func(ctx context.Context, uc *UnpackConfig) error {
+		uc.ApplyOpts = append(uc.ApplyOpts, opts...)
+		return nil
+	}
+}
+
```

Comment:
Don't know about any immediate need, but good to know it exists 👍🏻

Reply:
Yeah, it seems to be to make saving the content more robust to prevent potential corruption, but I haven't read in-depth under what circumstances that happened (so if we should unconditionally set it?) Looks like they enabled it for CRI, so perhaps ...?

cc @dmcgowan in case you are more familiar with those changes.

Reply:
I worded myself poorly. It's definitely needed for https://github.com/moby/moby/pull/47552/files/ec19fd6fed8c556bac6b9e1b57b3f7c093d2511d#diff-4a995e863f39233613fa453fc142ae3ece9e7d669d08ad9190001f495832710d.